src/Reducer/data-reducer.js

```diff
@@ -0,0 +1,117 @@
+export const dataReducer = (state, { type, payload }) => {
```

Comment:
Could use enums.

Reply:
I need to learn more about enums, will do it once I am comfortable.